doc/README.Win32.md

```diff
@@ -174,6 +174,18 @@ with control-S.
 Visual Studio will then re-run CMake.  If that completes without errors,
 you can build with Build > "Build All".
 
+### Building libpcap - Using vcpkg ###
+
+You can download and install libpcap using the [vcpkg](https://github.com/Microsoft/vcpkg) dependency manager:
```

Comment:
This doesn't, as far as I know, include the Npcap driver or packet library, so it won't support packet capture; should that be noted?  (I.e. it's sufficient for reading existing capture files and building code that uses libpcap, but it's not sufficient to let you capture traffic on your machine's interfaces.)

Reply:
npcap is not currently in vcpkg, you can submit an issue in vcpkg or directly submit a PR to add a new port, so that we will make libpcap depend on npcap and will realize the packet capture function.